.vscode/settings.json

```diff
@@ -49,7 +49,12 @@
   "python.pythonPath": ".venv/bin/python",
   // test discovery is sluggish and the UI around running
   // tests is often in your way and misclicked
+  // set this to true if you want to be able to click
+  // to debug a test
   "python.testing.pytestEnabled": false,
```

Comment:
Don't you need to set this to `true` ?

Reply:
Yes but according to the comment above it was set to false for a reason (i.e. query performance) so I didn't set it to true by default

Reply:
right, but how do you run test to attach the debug to without ?